src/h5web/toolbar/controls/DomainSlider/ErrorMessage.tsx

```diff
@@ -0,0 +1,36 @@
+import { FiCornerDownRight } from 'react-icons/fi';
+import { MdSwapVert } from 'react-icons/md';
+import styles from './DomainTooltip.module.css';
+
+interface Props {
+  message: string;
+  fallback: string;
+  showSwapBtn?: boolean;
+  onSwap?: () => void;
+}
+
+function ErrorMessage(props: Props) {
+  const { message, fallback, showSwapBtn = false, onSwap } = props;
```

Comment:
I would send the `error` as props and put the call of `getBoundErrorProps` (that would be renamed) in this component.

For `minGreater`, you could add a new type of `BoundError` created at https://github.com/silx-kit/h5web/pull/568/files#diff-0bff5f3cf4d4a01448d5c83c2021ad2d214a71511c758e866be6bad0698205aeR49. You would then need to add the appropriate associated message in `getBoundErrorProps`.

But I won't push strongly for this, feel free to keep as it is.

Reply:
No, you're right, great idea!